pkg/quarterdeck/middleware/auth.go

```diff
@@ -156,7 +167,7 @@ func ContextFromRequest(c *gin.Context) (ctx context.Context, err error) {
 		return nil, ErrNoRequest
 	}
 
-	// Add access token to context
+	// Add access token to context (from either header or cookie using Authenticate middleware)
 	ctx = req.Context()
 	if token := c.GetString(ContextAccessToken); token != "" {
 		ctx = api.ContextWithToken(ctx, token)
```

Comment:
I'm pretty sure this is where the access token is fetched to set the token in the header to Quarterdeck. @pdeziel can you confirm? If we're getting the access token from the context; then the `Authenticate` middleware is setting the access token context and it's using `GetAccessToken` to do that, which fetches from the cookies … so this should work for a cookie authentication forwarded to Quarterdeck.